store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10104,4 +10104,9 @@ TODO: delete them permanently from here
     indexing         - Generate index
   </desc>
 </attr>
+
+<attr id="9030" name="zimbraCountAccountsEnabled" type="boolean" cardinality="single" optionalIn="globalConfig" since="11.0.0">
```

Comment:
@desouzas @dasiyogesh what id is good? 4095?
Related comment: https://github.com/Zimbra/zm-mailbox/pull/1456

Reply:
I have seen id upto 9029 in zimbrOS/zm-mailbox (https://github.com/ZimbraOS/zm-mailbox/blob/develop/store/conf/attrs/zimbra-attrs.xml) , So next i took.

Reply:
id changed to 4096. Thanks